### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Storage Container_Keyboard: When the focus lands on the first item in the table, user is unable to select it immediately using Enter/space key

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3577]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS2.1.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.1.1

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) Tab till Windows menu option-->Show View-->Other-->Expand Azure-->Azure Explorer pane press OK
6) In Azure Explorer section, tab till HDInsight and expand it. Expand the cluster name
7) Tab till the storage container and press enter
8) In Storage Container Section, when focus lands on the first table item when it is non selected state, verify if user can be able to press enter/Space to select it

**Actual Result:** 
When the focus lands on the first item in the table user is unable to select it immediately using Enter/space key, apparently the user has to hit down arrow and then hit enter/space key to select the item.

**Expected Result:** 
In Storage Container Section, when focus lands on the first table item user should be able to select it with enter/space key right away when it is non selected state, or when the focus lands on the table element it should get automatically focused.

**User Impact:** 
Keyboard users will find difficulty in understanding how to select a table item if enter/space key cannot be used to select a table item

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_7_ New.pptx](https://github.com/microsoft/azure-tools-for-java/files/3640746/Bug_7_.New.pptx)

**Comment by t-rufang** [comment]
Note: Row of the table can't be selected with Enter/Space.
![image](https://user-images.githubusercontent.com/32627233/65828511-344b4800-e2ce-11e9-8788-14d5477b2ed0.png)

**Comment by Flanker32** [comment]
@Tanushagottipati We have removed storage explorer in IDE toolkit, could you please help us verify and close the issue? Here is the [build](https://dev.azure.com/mseng/VSJava/_build/results?buildId=11079936)

**Comment by Tanushagottipati** [comment]
As mentioned storage accounts tab cannot be opened in the eclipse toolkit. As storage explorer has been deprecated, closing this bug and attached screenshot for reference.
Environment:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-09 R (4.13.0)
Build id: 20190917-1200
Solution Installed: The Azure Toolkit for Eclipse 3.20.0
![Screenshot (709)](https://user-images.githubusercontent.com/41109135/70886781-59647280-2002-11ea-8f0e-7403782e4ac3.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749681
